### DISCUSSION TITLE
`Enter` in find(ctrl-f) mode should do "find next", like most other software.

### DISCUSSION THREAD

**Discussion by sisrfeng** [openstyles/stylus#1521] [Ideas]
The behavior of `Enter` key in find(ctrl-f) mode sometimes add newline to the code. It feels so strange.

**Comment by tophf** [comment]
Fixed in 075f796f. Now the Enter key will repeat search/replace same as in other apps.

**Comment by sisrfeng** [comment]
Thx!
When can the chrome extension be updated? How to update?
